projects/cdk/schematics/ng-update/steps/replace-identifier.ts

```diff
@@ -38,13 +38,29 @@ export function replaceIdentifier({from, to}: ReplacementIdentifier): void {
 
         if (Node.isImportSpecifier(parent)) {
             removeImport(parent);
-            addUniqueImport(
-                parent.getSourceFile().getFilePath(),
-                to.namedImport || to.name,
-                to.moduleSpecifier,
-            );
+            addImports(to, parent.getSourceFile().getFilePath());
         } else {
-            ref?.replaceWithText(to.name);
+            Array.isArray(to);
+            ref?.replaceWithText(
+                Array.isArray(to) ? to.map(({name}) => name).join(', ') : to.name,
+            );
         }
     });
 }
+
+function addImports(
+    identifier: ReplacementIdentifierMulti['to'],
+    filePath: string,
+): void {
+    if (Array.isArray(identifier)) {
+        identifier.forEach(({name, namedImport, moduleSpecifier}) => {
+            addUniqueImport(filePath, namedImport || name, moduleSpecifier);
+        });
+    } else {
+        addUniqueImport(
+            filePath,
+            identifier.namedImport || identifier.name,
+            identifier.moduleSpecifier,
+        );
+    }
+}
```

Comment:
```suggestion
function addImports(
    identifier: ReplacementIdentifierMulti['to'],
    filePath: string,
): void {
    return Array.isArray(identifier)
        ? identifier.forEach(x => addImports(x, filePath))
        : addUniqueImport(
              filePath,
              identifier.namedImport || identifier.name,
              identifier.moduleSpecifier,
          );
}
```

What do you think about this approach ? 🤔